src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -27,7 +27,7 @@
             <xsd:element name="property-access" type="property_access" minOccurs="0" maxOccurs="1" />
             <xsd:element name="scheduler" type="scheduler" minOccurs="0" maxOccurs="1" />
             <xsd:element name="serializer" type="serializer" minOccurs="0" maxOccurs="1" />
-            <xsd:element name="type-info" type="type_info" minOccurs="0" maxOccurs="1" />
+            <xsd:element name="type-info" type="type-info" minOccurs="0" maxOccurs="1" />
```

Comment:
we have much more cases using underscores in names of XSD types, so I would keep the name `type_info` (also, it looks like you did not remove `type_info` when adding `type-info`, which means you probably have dead code in the schema)